lib/plugins/create/templates/aws-nodejs-typescript/serverless.yml

```diff
@@ -10,9 +10,6 @@ provider:
   runtime: nodejs6.10
 
 functions:
-  # Example with LAMBDA-PROXY integration
-  # Invoking locally:
-  # sls webpack invoke -f hello
```

Comment:
We don't need this command anymore because [serverless-webpack is now integrated with `sls invoke local`](https://github.com/serverless-heaven/serverless-webpack/issues/151)